dask/graph_manipulation.py

```diff
@@ -6,7 +6,7 @@
 
 import uuid
 from collections.abc import Callable, Hashable
-from typing import Callable, Hashable, Literal, TypeVar
+from typing import AbstractSet, Any, Callable, Hashable, Literal, TypeVar
```

Comment:
please use collections.abc.Set instead

Reply:
nope. That's fed from HighLevelGraph.dependencies, which (very much deliberately) states that everything must be read-only.